db/migrate/20220225054741_create_plants.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePlants < ActiveRecord::Migration[5.2]
+  def change
+    create_table :plants do |t|
+      t.string :name
```

Comment:
we may need to define the relationship between plants and users. Plants should belong to a user.